plutus-chain-index/src/Plutus/ChainIndex/Emulator/DiskState.hs

```diff
@@ -102,11 +102,8 @@ fromTx tx =
 diagnostics :: DiskState -> Diagnostics
 diagnostics DiskState{_DataMap, _ValidatorMap, _MintingPolicyMap, _TxMap, _StakeValidatorMap, _RedeemerMap, _AddressMap} =
```

Comment:
Happened to notice `_DataMap` isn't used; maybe it should not be listed? Or does it make sense to list it in the diagnostics somewhere? (I note this isn't a change that's being made in this PR.)

Reply:
Thanks, I'll take a look at this in Part 2.